fontforgeexe/bitmapview.c

```diff
@@ -713,6 +712,16 @@ static void BVExpose(BitmapView *bv, GWindow pixmap, GEvent *event ) {
     RefChar *refs;
     extern Color widthcol;
 
+    CharView cvtemp;
+    CharViewTab cvtabtemp;
+    GGadgetData ggdatatemp;
+    GTabInfo gtitemp[2];
+    memset(gtitemp,0,sizeof(gtitemp));
+    gtitemp[0].text = (unichar_t *) '\0';
+    gtitemp[0].text_is_1byte = true;
+    ggdatatemp.u.tabs = gtitemp;
+    GGadget *ggtemp = GTabSetCreate(pixmap, &ggdatatemp, NULL);
```

Comment:
Not a fan of this approach to be honest, it's a pretty big hack to create a tab set just for the purposes of getting a 0 out of `GTabSetGetSel`. You're also missing freeing this.

My suggestion, which I was pondering this morning, is more along the lines of:

* Create a function, e.g. `CVGetActiveTab` that returns you a pointer to the tab data for the active tab. For the bitmap view, it sees that `tabs` is NULL and returns a pointer to the first element, something like:


```c
CharViewTab* CVGetActiveTab(CharView *cv) {
    if (cv->tabs == NULL) {
        // This happens when used by the bitmap view, which doesn't support tabs
        return &cv->cvtabs[0];
    }
    int tab = GtabSetGetSel(cv->tabs);
    return &cv->cvtabs[tab];
}
```

Reply:
Freeing it in BVExpose causes a segmentation fault because later code uses it :-)

Reply:
That makes me like this approach even less. What's holding onto the tabset after this 'temporary' charview goes out of scope?

Reply:
I think CharViewFree will clean it up

Reply:
But I do like your approach. I'm happy to change to using it. One moment please.